src/plugin/notification/index.ts

```diff
@@ -14,7 +15,11 @@ export const loadNotificationPlugin = async (
 ): Promise<NotificationPlugin> => {
   switch (config.type) {
     case ENotificationPluginKind.Discord:
-      return (await import("./discord")).get_plugin(config, HOST_INFO);
+      return (await import("./discord")).getPlugin(config, HOST_INFO);
+    case ENotificationPluginKind.Telegram:
+      return await (
+        await import("./telegram")
+      ).getPlugin(config as TelegramPluginConfiguration);
     default:
       throw new Error(`Plugin ${config.type} not supported!`);
```

Comment:
We should try lazy import by `id` and return `not found or invalid` if it fails.